dubbo-metrics/dubbo-metrics-api/src/main/resources/META-INF/dubbo/internal/org.apache.dubbo.rpc.cluster.filter.ClusterFilter

```diff
@@ -0,0 +1,2 @@
+observationreceiver=org.apache.dubbo.metrics.filter.observation.ObservationReceiverFilter
```

Comment:
See the comments below I made on Filter and ClusterFilter

Reply:
This needs to be changed, because `ObservationReceiverFilter` is implements `Filter`.
As shown in the figure, put it in the configuration file of `org.apache.dubbo.rpc.Filter`

Reply:
> This needs to be changed, because `ObservationReceiverFilter` is implements `Filter`. As shown in the figure, put it in the configuration file of `org.apache.dubbo.rpc.Filter`

Would implementing `org.apache.dubbo.rpc.ClusterFilter` be better?

Reply:
`org.apache.dubbo.rpc.ClusterFilter` should be applied to consumer, but `ObservationReceiverFilter` is provider.
So `ObservationReceiverFilter` shoule implement `Filter`